constants/src/main/java/org/jboss/pnc/constants/Patterns.java

```diff
@@ -53,5 +53,10 @@
      * the repository URL.
      */
     public static final String INTERNAL_REPOSITORY_NAME = "(\\/[\\w\\.:\\~_-]+)+(\\.git)(?:\\/?|\\#[\\d\\w\\.\\-_]+?)$";
-    
+
+    /**
+     * Represents an key in attribute of {@link org.jboss.pnc.model.ProductVersion}. It's value is
+     * tag prefix in Brew for a Version.
+     */
+    public static final String ATTRIBUTE_KEY_BREW_TAG_PREFIX = "BREW_TAG_PREFIX";
```

Comment:
This is not a regex pattern, move it to an appropriate class.

Reply:
Nice catch! Will do.